src/f32-vbinary/gen/f32-vmax-hvx-u64.c

```diff
@@ -0,0 +1,65 @@
+// Auto-generated file. Do not edit!
+//   Template: src/f32-vbinary/vop-hvx.c.in
+//   Generator: tools/xngen
+//
+
+#include <assert.h>
+
+#include <hvx_hexagon_protos.h>
+#include <hexagon_protos.h>
+#include <hexagon_types.h>
+
+#include <xnnpack/common.h>
+#include <xnnpack/math.h>
+#include <xnnpack/vbinary.h>
+
+void xnn_f32_vmax_ukernel__hvx_u64(
+    size_t batch,
+    const float* input_a,
+    const float* input_b,
+    float* output,
+    const union xnn_f32_default_params params[restrict XNN_MIN_ELEMENTS(1)]) XNN_OOB_READS
+{
+  assert(batch != 0);
+  assert(batch % sizeof(float) == 0);
+  assert(input_a != NULL);
+  assert(input_b != NULL);
+  assert(output != NULL);
+
+
+  const HVX_Vector *vptr_a = (const HVX_Vector *) input_a;
+  const HVX_Vector *vptr_b = (const HVX_Vector *) input_b;
+  HVX_Vector *vptr_o = (HVX_Vector*) output;
+
+  for (; batch >= 64 * sizeof(float); batch -= 64 * sizeof(float)) {
+    HVX_Vector va0 = *vptr_a++;
```

Comment:
Is this the best way to load on HVX?
Would it be better to load all va values and then all vb values?
Would it be better to use offsets and then increment the pointers after?
Are there LDP or LD4 instructions like ARM?

Reply:
I changed to load all va values then vb values for now. I will improve this part later using special instructions or offset.